docs/meshstack.user-group-provisioning.md

```diff
@@ -10,6 +10,16 @@ in the external system are automatically mirrored to the provisioned equivalents
 the members of a certain AAD group are changing and provisioning is set up, those changes will be reflected in the group within meshStack as well
 without additional effort. That way we can model the complete lifecycle of users and groups automatically.
 
+## Enable SCIM
+
+By default, in meshStack, you can create users through a diverse range of methods like the Panel, IDP, and other available
```

Comment:
By default, in meshStack, users can be created using a variety of methods such as the Panel, IDP, and other available options. If you prefer to manage your users exclusively via SCIM, you need to enable this setting in the Admin area. This disables all other methods to prevent conflicts and double assignments. To enable SCIM, go to the Settings page in the Admin area, and in the 'Manage' tab, turn on the 'SCIM only user management' button. By default, this button is turned off, allowing users and groups to be created through other means."

Reply:
Switched it to your text and also updated the screenshot.